src/localization/locales/en-US.json

```diff
@@ -16,7 +16,12 @@
     "group": {
         "create": "Create a Group",
         "home_title": "Groups",
-        "home_default_title": "Group",
+        "home_default_title": "Groups",
+        "img": {
+            "src": "groups.png"
+        },
+        "aria_label": "groups-list-title",
+        "aria_label_default": "groups-default-title",
```

Comment:
Issue: I don't think you need these 2, since you can use the IDs directly, they should be the same for all locales
```suggestion
```